Gemfile

```diff
@@ -11,7 +11,7 @@ gemspec
 # your gem to rubygems.org.
 
 # Shift Commerce gem
-gem "flex_commerce_api", git: "https://github.com/flex-commerce/flex-ruby-gem.git", tag: 'v0.6.29'
+gem "flex_commerce_api", git: "https://github.com/flex-commerce/flex-ruby-gem.git", tag: 'v0.6.45'
```

Comment:
I am afraid, in order to use this version of gem, FE rails-site code needs some changes. Safe version to use is `v0.6.43`

Reply:
Cheers @praWeb, I have downgraded to 0.6.43.